CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
```

Comment:
If we want to maintain such a file we should start using something like [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/). We can enforce this using `commitlint`